app/src/main/java/ca/llamabagel/transpo/ui/trips/GetNextBusTripsUseCase.kt

```diff
@@ -31,9 +32,9 @@ class GetNextBusTripsUseCase @Inject constructor(private val repository: TripsRe
             .flowOn(dispatcherProvider.main)
 
     private fun transformResponse(response: ApiResponse): List<TripAdapterItem> =
-            response.routes
-                .flatMap { route ->
-                    route.trips.map { trip -> TripItem(route, trip) }
-                }
-                .sortedBy { (_, trip) -> trip.adjustedScheduleTime }
+        response.routes
+            .flatMap { route ->
```

Comment:
Style wise, is it better to have:
``` kotlin
private fun transformResponse(response: ApiResponse): List<TripAdapterItem> = response.routes
         .flatMap { route ->
```

or put it all in the body like you did here?

Reply:
When using expression bodies that can't fit on one line, it's best to put it on the next line.

Reply:
But you can fit `response.routes` on the same line.